examples/storybook/src/stories/VoiceSearch.stories.ts

```diff
@@ -0,0 +1,158 @@
+import { storiesOf } from '@storybook/angular';
+import { wrapWithHits } from '../wrap-with-hits';
+import meta from '../meta';
+
+storiesOf('VoiceSearch', module)
+  .addDecorator(meta)
+  .add('default', () => ({
+    component: wrapWithHits({
+      template: `
+        <p>To see this button disabled, test it on unsupported browsers like Safari, Firefox, etc.</p>
+        <ais-voice-search></ais-voice-search>
+      `,
+    }),
+  }))
+  .add('without status', () => ({
+    component: wrapWithHits({
+      template: `
+        <ais-voice-search>
+          <ng-template #status></ng-template>
+        </ais-voice-search>
+      `,
+    }),
+  }))
+  .add('with a ais-search-box', () => ({
+    component: wrapWithHits({
+      template: `
+        <ais-voice-search></ais-voice-search>
+        <ais-search-box></ais-search-box>
+      `,
+    }),
+  }))
+  .add('with a custom button text', () => {
+    const style = window.document.createElement('style');
+    window.document.head.appendChild(style);
+    [
+      `.custom-button .ais-VoiceSearch-button:hover {
+        background: inherit;
+      }`,
+    ].forEach(rule => (style.sheet as CSSStyleSheet).insertRule(rule));
```

Comment:
I think it might be worth it to expose the [`style` property](https://angular.io/guide/component-styles#using-component-styles) that can be added to an angular component. Have you considered that?

Reply:
Thanks for pointing that out!
https://github.com/algolia/angular-instantsearch/pull/530/commits/3c75a6dd04b60c53dc01572bd88a7b606a9c8da1

Reply:
I didn't know it exists. Now that I'm considering it, and I guess I'm just going to move those styles to `styles.css` in `.storybook`. What do you think?
They're just some custom styles to make the stories look better.

Reply:
yes, we should probably expose both `style`  & `class` of the inner component

Reply:
I'm not following. 

1. exposing `style`
do you mean we provide a property like this,

```js
<ais-voice-search
  [style]="[
    `h1 { color: black }`,
    `h2 { color: grey }`
  ]"
>
<ais-voice-search/>
```

then, it's the same effect with

```js
@component({
  styles: [
    `h1 { color: black }`,
    `h2 { color: grey }`
  ]
})
class ...
```

If it's what you meant, I don't know how to do it. I googled it but couldn't find a proper keyword to get what I want.

2. exposing `class`

do you mean something like this we have in vue ?

```js
<ais-voice-search
  :class-names="{
    'ais-VoiceSearch': 'MyVoiceSearch',
    'ais-VoiceSearch-button': 'MyVoiceSearchButton',
    'ais-VoiceSearch-status': 'MyVoiceSearchStatus'
  }"
/>
```

Reply:
My  suggestion wasn't particular to this component though, so it can be changed later :)

Reply:
Yeah, nevermind @eunjae-lee  we can do that later. This works and is not so much harder to read.